### PR TITLE
Fixed bug when several files have different history length.

### PR DESCRIPTION
**Pull request by luixo**

### CODE REVIEW

**Comment by sindresorhus** [comment]
Tests are failing: https://travis-ci.org/sindresorhus/gulp-revert-path/builds/113169345

**Comment by luixo** [comment]
Fixed for 0.1x node test pass.

File: index.js [hunk 1]
```diff
@@ -7,12 +7,12 @@ module.exports = function (reversions) {
 	return through.obj(function (file, enc, cb) {
 		var history = file.history;
 		var highestIndex = history.length - 1;
+		var localReversions = reversions;
 
-		if (reversions > highestIndex) {
-			reversions = highestIndex;
-		}
+		if (localReversions > highestIndex)
+			localReversions = highestIndex;
```
Comment (sindresorhus) [inline index.js:13]: Don't change code style in PRs ;)

**Comment by sindresorhus** [comment]
Thank you :)

File: index.js (hunk 1 above)
Comment (luixo) [inline index.js:13]: I'll take note of that :)

**Pull request closed without merging**